games-engines/scummvm/scummvm-1.8.1.recipe

```diff
@@ -0,0 +1,96 @@
+SUMMARY="Script Creation Utility for Maniac Mansion Virtual Machine"
+DESCRIPTION="ScummVM is a program which allows you to run certain classic \
+graphical point-and-click adventure games, provided you already have their \
+data files. The clever part about this: ScummVM just replaces the executables \
+shipped with the games, allowing you to play them on systems for which they \
+were never designed!"
+HOMEPAGE="http://scummvm.org"
+SOURCE_URI="http://scummvm.org/frs/scummvm/$portVersion/scummvm-$portVersion.tar.xz"
+CHECKSUM_SHA256="935f53d7e4c460fbc22c58fee5007d4858891e4961831f53f575a3594b7f612a"
+REVISION="1"
+LICENSE="GNU GPL v2"
+COPYRIGHT="2001-2014 ScummVM Team"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+ADDITIONAL_FILES="scummvm.rdef"
+
+PROVIDES="
+	scummvm$secondaryArchSuffix = $portVersion
+	app:scummvm$secondaryArchSuffix = $portVersion
+"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libfaad$secondaryArchSuffix
+	lib:libflac$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libfluidsynth$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
+	lib:libglu$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libmad$secondaryArchSuffix
+	lib:libmpeg2$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libSDL$secondaryArchSuffix
+	lib:libtheora$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libfaad$secondaryArchSuffix
+	devel:libflac$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libfluidsynth$secondaryArchSuffix
+	devel:libgl$secondaryArchSuffix
+	devel:libglu$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libmad$secondaryArchSuffix
+	devel:libmpeg2$secondaryArchSuffix
+	devel:libogg$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	devel:libSDL$secondaryArchSuffix
+	devel:libtheora$secondaryArchSuffix
+	devel:libvorbis$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+"
+
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:git
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize
```

Comment:
It's safer to use **`cmd:libtoolize$secondaryArchSuffix`** (instead of just **`cmd:libtoolize`**). BTW, on some recipes you really need to add the **`$secondaryArchSuffix`** suffix for **`cmd:libtoolize`**, so, IMHO, it is better to use it for all recipes that have a SECONDARY_ARCHITECTURES.